src/contrib/persistence/Akka.Persistence.Sqlite/sqlite.conf

```diff
@@ -31,6 +31,14 @@
 			# timestamp provider used for generation of journal entries timestamps
 			timestamp-provider = "Akka.Persistence.Sql.Common.Journal.DefaultTimestampProvider, Akka.Persistence.Sql.Common"
 			
+			# The isolation level of all database read query
+			# Valid values: "chaos", "read-committed", "read-uncommitted", "repeatable-read", "serializable", "snapshot", or "unspecified"
+			read-isolation-level = unspecified
```

Comment:
These new isolation level settings need to be propagated to all `Sql.Common` implementation when they are upgraded to the next `Akka.NET` release.

Reply:
The differences are covered here: https://learn.microsoft.com/en-us/dotnet/api/system.data.isolationlevel?view=net-8.0#fields